### ISSUE TITLE
What is the latest version of Dasher for Windows?

### ISSUE THREAD

**Issue by smoothsaling** [dasher-project/dasher#170]
I am a user of Dasher, not a coder and my knowledge of GitHub is razor thin. I worked with Ada Majorek in 2015&16 while she updated Dasher and released Dasher 5.0. I have some of Ada's Google engineer colleagues who want to improve Dasher's prediction with some new machine learning algorithms. Because I am largely ignorant of GitHub, can someone confirm that the latest source code is located at the following link? If not, please provide me a link to the latest, most updated source code of Dasher for Windows? Thank you.
https://github.com/dasher-project/dasher/releases

@cagdasgerede
@willwade
@jcope

**Comment by willwade** [comment]
Hi Steve. Thanks for getting in touch. Well to answer your question - yes, as far as I know this is the latest - but do bear in mind this is totally open sourced so anyone anywhere could have forked and be working on it currently. Great timing though Steve - we have a developer working on a new Dasher - totally from the ground up. Its very early days - but our current big question is language models and how we link these in - in a modular way. We have some chat with the original Dasher team - and the developer on our slack channel currently - if you want to get involved (and please do!) - drop me a line at wwade@acecentre.org.uk and I'll get you - and the google team - onboard.

**Comment by sebwills** [comment]
My understanding is that this repo is Ada's fork of the gnome Dasher repo, but renamed and re-homed in the dasher-project github organisation. Which is why the Dasher 5 link on the inference group site to Ada's repo, https://github.com/ipomoena/dasher , redirects here. In other words, this repo is the latest 'official' Dasher 5 source code.

@cagdasgerede can you confirm this is correct?

**Comment by cagdasgerede** [comment]
That is correct. I asked Ada's husband to redirect Ada's repo address to this space.
